regoservice/eval.go

```diff
@@ -183,15 +183,47 @@ func BuildBundledRegoEvaluator(rego_package string) (data.Inpteval, error) {
 		aggrQuery:        aggrQuery,
 	}, nil
 }
+
+// loadsFuncs try to load rego files from the filesystem,
+// in case the file paths are not found it loads the regos from the embedded FS
+func loadFuncs(templatesDir []string, f func() []func(r *rego.Rego)) []func(r *rego.Rego) {
+	foundPaths := make([]string, 0)
+	for _, path := range templatesDir {
+		if _, err := os.Stat(path); !os.IsNotExist(err) {
+			foundPaths = append(foundPaths, path)
+		}
+	}
+
+	if len(foundPaths) != 0 {
+		return []func(r *rego.Rego){rego.Load(foundPaths, nil)}
+	}
+
+	return f()
+}
+
+func buildTemplatesFuncs() (funcs []func(r *rego.Rego)) {
```

Comment:
`buildTemplatesFuncs()` is almost same with `buildCommonFuncs()`. It can be replaced with something like `buildModuleFuncs(map[string] string) funcs []func(r *rego.Rego)`